client/tx/tx.go

```diff
@@ -201,11 +201,28 @@ func Sign(txf Factory, name string, txBuilder client.TxBuilder, overwriteSig boo
 	if err != nil {
 		return err
 	}
+
 	pubKey := key.GetPubKey()
+	tx1 := txBuilder.GetTx()
```

Comment:
nit: Why call this `tx1`? can we call this just `tx` or something more meaningful?